playbooks/PnP.yml

```diff
@@ -0,0 +1,53 @@
+---
+- name: Manage operations add device, claim device and unclaim device of Onboarding Configuration(PnP)
+  hosts: localhost
+  connection: local
+  gather_facts: no
+
+  vars_files:
+    - "{{ CLUSTERFILE }}"
+
+  vars:
+    dnac_login: &dnac_login
+      dnac_host: "{{ dnac_host }}"
+      dnac_username: "{{ dnac_username }}"
+      dnac_password: "{{ dnac_password }}"
+      dnac_verify: "{{ dnac_verify }}"
+      dnac_port: "{{ dnac_port }}"
+      dnac_version: "{{ dnac_version }}"
+      dnac_debug: "{{ dnac_debug }}"
+
+  tasks:
+
+    - name: Add a new device and claim the device
```

Comment:
The task is adding a new device but the state is mentioned as deleted. Can you please check?